src/UI/Art/SpriteArtist/PlayerSetupArtist.java

```diff
@@ -0,0 +1,270 @@
+package UI.Art.SpriteArtist;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.util.HashMap;
+
+import Terrain.MapInfo;
+import UI.PlayerSetupController;
+import UI.PlayerSetupInfo;
+import UI.UIUtils;
+import Units.UnitModel;
+
+public class PlayerSetupArtist
+{
+  private static final Color MENUFRAMECOLOR = new Color(169, 118, 65);
+  private static final Color MENUBGCOLOR = new Color(234, 204, 154);
+  private static final Color MENUHIGHLIGHTCOLOR = new Color(246, 234, 210);
+
+  private static double animHighlightedPlayer = 0;
+
+  private static PlayerSetupController myControl = null;
+
+  private static HashMap<Integer, PlayerPanel> playerPanels = new HashMap<Integer, PlayerPanel>();
```

Comment:
I have to wonder: why not just an `ArrayList`?

Reply:
Uhhh. That's what my brain decided to type. ArrayList would work fine I think, though it would change the implementation slightly. Apparently I like maps.